downstreamadapter/dispatcher/dispatcher.go

```diff
@@ -49,7 +50,20 @@ type EventDispatcher interface {
 	GetSyncPointInterval() time.Duration
 	GetStartTsIsSyncpoint() bool
 	GetResolvedTs() uint64
-	HandleEvents(events []DispatcherEvent, wakeCallback func()) (block bool)
+	HandleEvents(events []DispatcherEvent, wakeCallback func()) bool
```

Comment:
Why we need add these function in this interface? This interface just need to include some function used in Event Service, not all functions in dispatchers?

Reply:
Oh, if you want it's a interface for dispatcher and redoDispatcher, it's also better to use another interface, not just expand this one. It's better to make different interface for different use.